devito/arch/compiler.py

```diff
@@ -528,6 +528,60 @@ def __lookup_cmds__(self):
         self.MPICXX = 'mpicxx'
 
 
+class OneapiCompiler(Compiler):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        language = kwargs.pop('language', configuration['language'])
+        platform = kwargs.pop('platform', configuration['platform'])
+
+        self.cflags.append("-xHost")
+        self.cflags.append("-qopt-zmm-usage=high")
+
+        if configuration['safe-math']:
+            self.cflags.append("-fp-model=strict")
+        else:
+            self.cflags.append('-fast')
```

Comment:
-fp-model=fast like in IntelCompiler? or just -fast ?

Reply:
-fast here works fine, should be the same